packages/app-builder-lib/package.json

```diff
@@ -76,6 +76,7 @@
     "minimatch": "^10.0.3",
     "plist": "3.1.0",
     "resedit": "^1.7.0",
+    "resolve": "^1.22.10",
```

Comment:
Why do we need a browserify's `resolve` module? From their repo:
> implements the [node require.resolve() algorithm](https://nodejs.org/api/modules.html#modules_all_together) such that you can require.resolve() on behalf of a file asynchronously and synchronously

Is there any benefit here to just using node's `require.resolve`?